web/server/www/scripts/codecheckerviewer/CommentView.js

```diff
@@ -158,6 +151,54 @@ function (declare, dom, style, topic, Memory, Observable, ConfirmDialog,
     }
   });
 
+  var SystemComment = declare(ContentPane, {
+    postCreate : function () {
+      var wrapper = dom.create('div', { class : 'wrapper'}, this.domNode);
+
+      dom.create('span', { class : 'system-comment-icon' }, wrapper);
+
+      // Create comment time.
+      var timeAgo = util.timeAgo(new Date(this.commentData.createdAt.replace(/ /g,'T')));
+      var time = dom.create('span', { class : 'time', innerHTML: timeAgo });
+
+      // Create comment author.
+      var author = dom.create('span', {
+        class : 'author',
+        innerHTML: this.commentData.author
+      });
+
+      // Replace special string in comment message.
+      var message = this.commentData.message;
+      message = message
+        .replace('%author%', author.outerHTML)
+        .replace('%date%', time.outerHTML);
+
+      var divMsg = dom.create('div', {
+        class : 'message',
+        innerHTML: message
+      }, wrapper);
+
+      this.addStatusIcons(divMsg);
+    },
+
+    addStatusIcons : function (domElement) {
+      ['detection-status', 'review-status'].forEach(function (type) {
+        ['old-' + type, 'new-' + type].forEach(function (className) {
+          var domStatuses =
+            domElement.getElementsByClassName(className);
+
+          for (var i = 0; i < domStatuses.length; i++) {
+            var domStatus = domStatuses[i];
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable Assigned to Object Injection Sink (security/detect-object-injection)](https://app.codacy.com/app/CodeChecker/codechecker/pullRequest?prid=3374942)